tdrs-frontend/src/actions/reports.js

```diff
@@ -103,17 +113,14 @@ export const upload = ({ file, section }) => async (dispatch) => {
     dispatch({
       type: SET_FILE,
       payload: {
+        file: file,
         fileName: file.name,
         fileType: file.type,
         section,
         uuid: uuidv4(),
       },
     })
   } catch (error) {
-    dispatch({
```

Comment:
This seemed to be here just to satisfy tests, I found a different way to do the error assertion: https://github.com/raft-tech/TANF-app/pull/1061/files#diff-0055c364941646ce44cc90d151c9ff4b46c27f25863c396715202e5478b65a89R61-R64